src/lib/process/index.ts

```diff
@@ -68,11 +72,28 @@ export const createProcess = async (
       program,
     }
 
-  const process: Process.Payload = await createProcessTransaction(polkadot, processId, program, options)
+  if (version === currentVersion) {
```

Comment:
I think all validation should happen before `dryRun` handling so we can properly test for success using `dryRun`

Reply:
changed back.